### PR TITLE
[CI:DOCS] troubleshooting: document keep-id options

### PR DESCRIPTION
**Pull request by eriksjolund**
Add new troubleshooting tip:
Podman run fails with "Error: unrecognized namespace mode keep-id:uid=1000,gid=1000 passed"

Update the troubleshooting tips:
"Passed-in devices or files can't be accessed in rootless container (UID/GID mapping problem)" and
"Container creates a file that is not owned by the user's regular UID" to use
"--userns keep-id:uid=$uid,gid=$gid"
instead of the command-line options --uidmap and --gidmap

Signed-off-by: Erik Sjölund <erik.sjolund@gmail.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
Thanks @eriksjolund 
LGTM
@giuseppe PTAL

File: troubleshooting.md [hunk 1]
```diff
@@ -1291,3 +1280,67 @@ $ podman run --rm -t fedora /bin/sh -c "stty -onlcr && echo abc" | od -c
 0000000   a   b   c  \n
 0000004
 ```
+
+### 39) Podman run fails with "Error: unrecognized namespace mode keep-id:uid=1000,gid=1000 passed"
+
+Podman 4.3.0 introduced the options _uid_ and _gid_ that can be given to `--userns keep-id`.
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md:1286]:
```suggestion
Podman 4.3.0 introduced the options _uid_ and _gid_ that can be given to `--userns keep-id` which are not recognized by older versions of Podman.
```

File: troubleshooting.md [hunk 2]
```diff
@@ -1291,3 +1280,67 @@ $ podman run --rm -t fedora /bin/sh -c "stty -onlcr && echo abc" | od -c
...
+
+#### Symptom
+
+When using a Podman version older than 4.3.0, the options _uid_ and _gid_ are not recognized.
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md:1290]:
```suggestion
When using a Podman version older than 4.3.0, the options _uid_ and _gid_ are not recognized, and an "unrecognized namespace mode" error is raised.
```

File: troubleshooting.md [hunk 3]
```diff
@@ -1291,3 +1280,67 @@ $ podman run --rm -t fedora /bin/sh -c "stty -onlcr && echo abc" | od -c
...
+  --user $uid:$gid \
+  --userns keep-id:uid=$uid,gid=$gid \
+     docker.io/library/ubuntu /bin/cat /proc/self/uid_map
+Error: unrecognized namespace mode keep-id:uid=1000,gid=1000 passed
+$
+```
+
+#### Solution
+
+Use __--uidmap__ and __--gidmap__ to describe the same UID and GID mapping.
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md:1305]:
```suggestion
Use __--uidmap__ and __--gidmap__ options to describe the same UID and GID mapping.
```

**Comment by eriksjolund** [comment]
@TomSweeneyRedHat thanks for the suggestions. Now applied.

**Comment by rhatdan** [comment]
LGTM
@TomSweeneyRedHat @giuseppe PTAL

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM
Thx @eriksjolund !

**Pull request merged**